lib/tether/tether.go

```diff
@@ -731,12 +751,6 @@ func (t *tether) launch(session *SessionConfig) error {
 		session.Cmd.SysProcAttr = user
 	}
 
-	err = t.loggingLocked(session)
```

Comment:
instead of removing this and adding above, I believe you could simply do following here;

```go
if session.Diagnostics.ResurrectionCount > 0 {
    err = t.loggingLocked(session)
....
}
```